tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java

```diff
@@ -109,6 +109,11 @@ public void journalSync() throws Exception {
     tableMaster.syncDatabase(DB_NAME);
     checkTable(tableMaster, DB_NAME, 2, 3);
 
+    // Drop a table to create a 'remove_table' entry
+    genTable(1, 3, true);
+    tableMaster.syncDatabase(DB_NAME);
+    checkTable(tableMaster, DB_NAME, 1, 3);
+
     restartMaster();
```

Comment:
@yuzhu I was expecting the old code to throw an exception (Unrecognized journal entry something) during restart, which doesn't happen. Does it run asynchronously?

Reply:
I remember seeing an exception being shown when running in intellij.  I am guessing it is a server side exception, which might not cause the test (client) to fail.. it might also get masked somehow.

Reply:
I see, thanks! I guess that means the PR is ready to merge?

Reply:
i am ok with this PR. @gpang do you want to take another look? also, do you know why a `Journal replay error` does not fail the test?  I don't want this issue to happen else where and not be caught.

Reply:
Yeah, that is strange. I don't know why that error doesn't fail the test. In any case, we can change that part of it in a different PR.